File: Java/src/jakopec/t3nasljedivanje/Start.java
package jakopec.t3nasljedivanje;

import javax.print.attribute.standard.NumberOfDocuments;

public class Start {

    public  Start(){


        Nova n = new Nova();
        System.out.println(n);




        Dokument dokument = new Dokument();
        dokument.setSifra(1);
        dokument.setBrojDokumenta("D1");
        dokument.setIznos(200);
        System.out.println(dokument);


        RadniNalog radniNalog = new RadniNalog();
        radniNalog.setSifra(1);
        radniNalog.setBrojDokumenta("RN1");
        radniNalog.setIznos(1200);
        radniNalog.setRadnik(new Osoba(1,"Marko","Markovina"));
        System.out.println(radniNalog);



        Racun racun = new Racun();
        racun.setSifra(1);
        racun.setBrojDokumenta("2020-001");
        racun.setIznos(1200.34);
        racun.setKupac("FFOS");
        System.out.println(racun);

    }

    public static void main(String[] args) {
        new Start();
    }
}

File: Kotlin/src/main/kotlin/t3nasljedivanje/Start.kt
package t3nasljedivanje

/**
 * @author Matija Skrletović
 **/

class Start {

    init {
        val n = Nova()
        println(n)
        val dokument = Dokument(1, "D1", 200.toDouble())
        println(dokument)
        val radniNalog = RadniNalog(1, "RN1", 1200.toDouble(), Osoba(1, "Marko", "Markovina"))
        println(radniNalog)
        val racun = Racun(1, "2020-001", 1200.34, "FFOS")
        println(racun)

    }
}

fun main() {
    Start()
}